service/controller/legacy/v29patch1/templates/cloudformation/tccp/auto_scaling_group.go

```diff
@@ -20,6 +20,7 @@ const AutoScalingGroup = `
       LaunchConfigurationName: !Ref {{ $v.ASGType }}LaunchConfiguration
       LoadBalancerNames:
         - !Ref IngressLoadBalancer
+        - !Ref IngressInternalLoadBalancer
```

Comment:
We should then port this to `v30` as well - I will take care.

Reply:
I can do it in a follow-up PR

1.